Mathlib/Data/Seq/Defs.lean

```diff
@@ -0,0 +1,729 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+import Mathlib.Data.Option.NAry
+import Mathlib.Data.Seq.Computation
+
+/-!
+# Possibly infinite lists
+
+This file provides a `Seq α` type representing possibly infinite lists (referred here as sequences).
```

Comment:
```suggestion
This file provides `Stream'.Seq α`, a type representing possibly infinite lists (referred here as sequences).
```

Optional, but would be much appreciated: this might also be a good chance to expand these module docs with Main definitions / Main statements and anything else you learned that might be useful for future users. cf. [our documentation guide](https://leanprover-community.github.io/contribute/doc.html#header-comment)